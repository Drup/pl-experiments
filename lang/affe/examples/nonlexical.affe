let ignore _ = ()

let f x =
  let y = &x in
  ignore y;
  &!x