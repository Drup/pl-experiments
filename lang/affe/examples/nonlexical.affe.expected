ignore : ∀^k, 'a. ('a : ^k) & (^k < affₙ) => 'a -> unit

Kind error: Cannot unify kinds un₄
and lin₃

